tasks/main.yml

```diff
@@ -1 +1,155 @@
-# role tasks
+# Tasks are insipred and named after the Red Hat OpenStack install directions
+- name: 4.2 Creating directories for templates and images
+  become: "{{ osp_director_become }}"
+  become_user: "{{ osp_director_become_user }}"
+  file:
+    path: "{{ item }}"
+    state: directory
+    mode: 0755
+  with_items:
+    - ~/images
+    - ~/templates
+  tags:
+    - "4.2"
+
+- name: 4.5 Installing the director packages
+  become: "{{ osp_director_become }}"
+  become_user: "{{ osp_director_become_user }}"
+  block:
+    - name: Install main Director packages
+      package:
+        name: "{{ osp_director_base_packages }}"
+        state: present
+
+    - name: install tempest-related packages
+      package:
+        name: "{{ osp_director_tempest_packages }}"
+        state: present
+      tags:
+        - tempest
+
+    - name: Add useful aliases for OpenStack
+      become: true
+      become_user: stack
+      blockinfile:
+        path: ~/.bashrc
```

Comment:
I read that there may be a better way to reference home dirs [here](https://github.com/oasis-roles/ocp_azure_driver/pull/1#discussion_r373098452).

Reply:
See previous comment.

Reply:
I commented up there, too, but on this specifically it's worth noting that `.bashrc` is not read by default when starting a login shell, so this should probably go in `.bash_profile`. As an alternative, though, I question why the role is messing around with user aliases at all, and propose that we don't do that.

Reply:
As for why we are doing it and where, that's because this is where and how the file is handled by @homeski 's group. I'd like to hear from him on why this is done thusly.

Reply:
I'm going to propose moving these to the stackrc file. That file is specific to openstack and isn't sourced by default when opening a shell.